### PR TITLE
Organization admin actions

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: data_api/staging/admin.py [hunk 1]
```diff
@@ -9,10 +21,23 @@ class SyncCheckpointAdmin(admin.ModelAdmin):
...
+    def _sync_organization(self, request, pk):
+        organization = get_object_or_404(Organization, pk=pk)
+        client = TembaClient(organization.server, organization.api_token)
+        import_org_with_client(client, organization.server, organization.api_token)
+        return HttpResponseRedirect(reverse('admin:staging_organization_change', args=[organization.pk]))
+
+    @action()
+    def _sync_latest_data(self, request, pk):
+        organization = get_object_or_404(Organization, pk=pk)
+        sync_latest_data(entities=None, orgs=[organization.api_token])
```
Comment (czue) [inline data_api/staging/admin.py]: I think we should call this asynchronously since it could take hours or longer to complete.

Comment (domdinicola) [reply data_api/staging/admin.py]: what do you mean with screenshot?
like a screenshot file in the repo?

**Comment by czue** [comment]
> what do you mean with screenshot?
> like a screenshot file in the repo?

Sorry, no just like pasted in the PR description. Like this:

image

(just using that as an example)

I guess it's a bit silly with the admin changes, but I'm not too familiar with how they work yet, so is helpful to see without having to run the code locally. It's a practice I've gotten to like from other projects whenever doing UI work.

**Comment by domdinicola** [comment]
Screenshot_2019-08-06_09-44-11

**Pull request merged**